qa/tasks/cephfs/test_data_scan.py

```diff
@@ -101,6 +101,16 @@ def scrub(self):
         self.assert_equal(out_json["return_code"], 0)
         self.assert_equal(self._filesystem.wait_until_scrub_complete(tag=out_json["scrub_tag"]), True)
 
+    def mangle(self):
+        """
+        Gives an opportunity to fiddle with metadata objects before bringing back
+        the MDSs online. This is used in testing the lost+found case (when recovering
+        a file without a backtrace) to verify if lost+found directory object can be
+        removed via RADOS operation and the file system can be continued to be used
+        as expected.
```

Comment:
Sorry, in which situation are we recommending users to delete (using `rados`) the lost+found directory? Your commit message says:

> Users are recommended to fail the filesystem and remove the directory object using rados cli commands.

I thought the issue to test was that a user could unlink or rename files normally inside the lost+found directory?

Reply:
> Sorry, in which situation are we recommending users to delete (using `rados`) the lost+found directory? Your commit message says:
> 
> > Users are recommended to fail the filesystem and remove the directory object using rados cli commands.
> 
> I thought the issue to test was that a user could unlink or rename files normally inside the lost+found directory?

This was broken and was fixed with commit 241b9396dbc6d506cef297bdb89ab07fe3a7ce02. So, users running older versions would have to cleanup via removing omap keys. That's what this test verifies.